src/style.css

```diff
@@ -1,3 +1,67 @@
+:root {
+  --text-color: rgba(0, 0, 0, 1);
+  --hover-color: rgba(0, 172, 220, 1);
+  --active-color: rgba(0, 172, 220, 1);
+}
+
 body {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: block;
+  padding: 23px 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  text-transform: uppercase;
+  color: var(--text-color);
+}
+
+:hover {
+  color: var(--hover-color);
+}
+
+.is-active {
+  position: relative;
+  color: var(--active-color);
+}
+
+::after {
```

Comment:
```suggestion
.is-active::after {
```

Reply:
.is-active has been added. 
c10b1a28bf020027d9a006e26005fd5aa9d96204